samples/tfm/provisioning_image_net_core/prj.conf

```diff
@@ -1,7 +1,9 @@
 #
-# Copyright (c) 2022 Nordic Semiconductor ASA
+# Copyright (c) 2023 Nordic Semiconductor ASA
```

Comment:
I don't think modifying an existing file should change the first year the file is copyrighted

Reply:
I reverted all the files back :)